AccumulatedPatterns[] accumulated_patterns # array since we have multiple sensors
---
